src/pages/PostPages/StepIndicator/style.css.ts

```diff
@@ -0,0 +1,31 @@
+import { css, cva } from "@styled-system/css";
+
+export const progressCircle = cva({
+    base: {
+        height: '0.375rem',
+        transition: 'all 0.3s ease-in-out'
+    },
+    variants: {
+        color: {
+            main: {
+                // width: '0.875rem',
+                backgroundColor: 'main',
+                borderRadius: '0.625rem',
+            },
+            gray: {
+                // width: '0.375rem',
+                backgroundColor: '54',
+                borderRadius: 'full',
+            }
+        }
```

Comment:
```suggestion
        },
        isCurrent: {
          true: {
            width: '0.875rem',
          },
          false: {
            width: '0.375rem',
          },
        },
```

여기도 isCurrent 이렇게 추가해서
`StepIndicator` 컴포넌트에서 이런식으로 제어하면
```ts
    <div className={s.Container}>
      {Array.from({ length: totalSteps }, (_, i) => {
        const stepNumber = i + 1;
        const isActive = stepNumber <= currentStep;
        const isCurrent = stepNumber === currentStep;

        return <div key={stepNumber} className={s.progressCircle({ isActive, isCurrent })} />;
      })}
    </div>
```
좀더 깔끔하기도 하고 성능적으로 조흠
왜 성능적으로 더 좋은지는 제로런타임CSS 검색고고곡 

<img width="974" alt="Screenshot 2025-07-04 at 4 06 24 PM" src="https://github.com/user-attachments/assets/e20665ee-9e10-4514-a3df-949e4dfbd6d4" />